neurots/generate/grower.py

```diff
@@ -346,7 +347,7 @@ def _grow_trunks(self):
                 )
                 n_trees = len(orientations)
 
-                if type_of_tree == "basal_dendrite" and n_trees < 2:
+                if type_of_tree == "basal" and n_trees < 2:
```

Comment:
this is rebase issue I think

Reply:
it's a bit to small to create a function, no?

Reply:
Yeah and when the 'legacy' part will be removed it will not make sense to have a function for it. But having a function ensures that both cases are handled the same way. So as you prefer, but they at least have to be the same.